lib/rules/selector-no-type/README.md

```diff
@@ -50,7 +50,25 @@ The following patterns are *not* considered warnings:
 
 ## Optional secondary options
 
-### `ignore: ["compounded", "descendant"]`
+### `ignore: ["child", "compounded", "descendant"]`
+
+#### `"child"`
+
+Allow child selectors.
+
+The following patterns are *not* considered warnings:
+
+```css
+a > foo {}
+```
+
+```css
+a > #bar {}
+```
+
+```css
+#bar > a {}
+```
```

Comment:
Either I'm not understanding the intention, or something is wrong here. In the first two cases, `a` is *not* a child, therefore it *should* be considered a warning, right? Only the last case has no type selectors that are not children. If the intention is that you always allow types in compound selectors that involve child combinators — I don't think I understand the justification for that. I *think* the original request in #1942 only addressed *type selectors following child combinators* — that is, types that are children.